tree/tree/test/BulkApiMultiple.cxx

```diff
@@ -25,27 +25,223 @@ class BulkApiMultipleTest : public ::testing::Test {
 
       // Otherwise, we keep with the current ROOT defaults.
       auto tree = new TTree("T", "A ROOT tree of floats.");
+      // bool, short, int, float, int64_t to work, while "char
       float f = 2;
       double g = 3;
+      bool b = true;
+      short s  = 4;
+      int i = 5;
+      Long64_t ll = (1<<20) + 1;
+      char c = 7;
```

Comment:
This leaves valgrind no chance, and might get valid looking values to users where in fact some I/O logic has failed. Consider leaving those uninitialized.

Reply:
Argh it's a test.